stt.c

```diff
@@ -362,5 +392,12 @@ case 'h':
 
 		timesnode_print(timestree, mktime(&today));
 	}
+
+    fclose (fp);
+
+    timesnode_free (timestree);
+
+    free_and_null (nowtime);
+
```

Comment:
Not really required on targeted systems, as the OS will take care of freeing remaining memory & resources